.github/workflows/static_checks.yml

```
@@ -114,3 +100,6 @@ jobs:
           skip: "./bin,./thirdparty,*.desktop,*.gen.*,*.po,*.pot,*.rc,./AUTHORS.md,./COPYRIGHT.txt,./DONORS.md,./core/input/gamecontrollerdb.txt,./core/string/locales.h,./editor/project_converter_3_to_4.cpp,./misc/scripts/codespell.sh,./platform/android/java/lib/src/com,./platform/web/node_modules,./platform/web/package-lock.json"
           ignore_words_list: "breaked,curvelinear,doubleclick,expct,findn,gird,hel,inout,lod,mis,nd,numer,ot,requestor,te,vai"
           path: ${{ env.CHANGED_FILES }}
+
+      - name: Other checks via pre-commit
+        uses: pre-commit/action@v3.0.0
```

```suggestion
        uses: pre-commit/action@v3.0.0

```
Missing line termination